frontend/src/hocs/withHostCommunication/withHostCommunication.test.tsx

```diff
@@ -24,13 +24,16 @@ import withHostCommunication, {
   HOST_COMM_VERSION,
 } from "./withHostCommunication"
 
-const TestComponentNaked = (props: {
+interface TestProps {
   hostCommunication: HostCommunicationHOC
-}): ReactElement => {
-  return <div>test</div>
+  label: string
```

Comment:
nit: would be useful to leave a comment mentioning that `label` is only used to ensure that the type checker is working as expected

Reply:
Cool - actually, i'll go further and change the name of the property to make its intent more clear